services/management/ia/templates_service.go

```diff
@@ -151,6 +162,126 @@ func (svc *TemplatesService) loadFile(ctx context.Context, file string) ([]saas.
 	return rules, nil
 }
 
+type ruleFile struct {
+	Group []ruleGroup `yaml:"groups"`
+}
+
+type ruleGroup struct {
```

Comment:
I think that type can be defined inside `dumpRule` func.

Reply:
any reason why you want that? we can do it, but that way `ruleFile` and `ruleGroup` would have to be moved inside `dumpRule` and I wanted all three types for the alert rule to stay in one place.

Reply:
Well, the scope of these structs just one function in fact. `ruleFile` and `ruleGroup` are used only for serialization. So it's nice to keep the scope as small as possible.

Reply:
will change it

Reply:
@artemgavrilov there is a small problem with that, since I use `ruleFile` in my tests I need it outside the `dumpRule` function

Reply:
Well, ok then. Leave it as is